test/e2e/scalability/density.go

```diff
@@ -567,9 +586,11 @@ var _ = SIGDescribe("Density", func() {
 						LogFunc:   framework.Logf,
 					})
 			}
-			e2eStartupTime = runDensityTest(dConfig)
+			e2eStartupTime = runDensityTest(dConfig, testPhaseDurations)
 			if itArg.runLatencyTest {
 				By("Scheduling additional Pods to measure startup latencies")
+				latencyMeasurementPhase := testPhaseDurations.StartPhase(800, "latency measurment")
```

Comment:
This is a bit too coarse-grained. Could you split further into at least the following phases:
- latency pods creation
- pod startup latency measurements
- latency pods deletion

Reply:
Done.